code/modules/mob/living/carbon/happiness_events.dm

```diff
@@ -0,0 +1,104 @@
+/datum/happiness_event
```

Comment:
oh this isnt a component, im not even sure what you're using this exactly like a component for

Reply:
i cant use components